### PR TITLE
refactor(db): remove openId

### LINKED ISSUE #142: Remove openId stuff
Removed in auth-server: https://github.com/mozilla/fxa-auth-server/pull/1337#discussion_r70516244

Let's try to delete the endpoints and maybe the db stuff.

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #142

### CODE REVIEW

File: lib/db/schema/patch-026-027.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
+DROP PROCEDURE `openIdRecord_1`;
+DROP PROCEDURE `createAccount_3`;
```
Comment (vladikoff) [inline lib/db/schema/patch-026-027.sql]: @rfk can this fallback to `createAccount_2` or do we need `createAccount_4`?

Comment (vladikoff) [inline lib/db/schema/patch-026-027.sql]: cc @jrgm

Comment (vladikoff) [inline lib/db/schema/patch-026-027.sql]: from mtg: "going backwards with versions seems wrong"

**Comment by philbooth** [comment]
I know this has @rfk's face on it but fwiw, other than the question about versioning the `createAccount` stored procedure, this LGTM. 👍

File: lib/db/schema/patch-026-027.sql (hunk 1 above)
Comment (rfk) [inline lib/db/schema/patch-026-027.sql]: I support "going backwards with versions seems wrong", let's make it `createAccount_4`.

**Comment by rfk** [comment]
> other than the question about versioning the createAccount stored procedure, this LGTM.

Yep, what @philbooth said :-)

**Comment by vladikoff** [comment]
@philbooth @rfk updated, r?

**Comment by philbooth** [comment]
**r+** (tests haven't finished in CI yet for some reason, but they're good locally)

**Comment by vladikoff** [comment]
Travis cannot install mysql:

```
0.53s$ sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install mysql-server-5.6 mysql-client-core-5.6 mysql-client-5.6
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package mysql-server-5.6
E: Couldn't find any package by regex 'mysql-server-5.6'
E: Unable to locate package mysql-client-core-5.6
E: Couldn't find any package by regex 'mysql-client-core-5.6'
E: Unable to locate package mysql-client-5.6
E: Couldn't find any package by regex 'mysql-client-5.6'
apt-get.diagnostics
apt-get install failed
$ cat ~/apt-get-update.log
Get:1 http://us-central1.gce.archive.ubuntu.com precise Release.gpg [198 B]
Get:2 http://us-central1.gce.archive.ubuntu.com precise-updates Release.gpg [198 B]
Get:3 http://us-central1.gce.archive.ubuntu.com precise-backports Release.gpg [198 B]
Get:4 http://us-central1.gce.archive.ubuntu.com precise Release [49.6 kB]
Get:5 http://us-central1.gce.archive.ubuntu.com precise-updates Release [55.4 kB]
Get:6 http://us-central1.gce.archive.ubuntu.com precise-backports Release [55.5 kB]
Get:7 http://us-central1.gce.archive.ubuntu.com precise/main Sources [934 kB]
Hit http://apt.postgresql.org precise-pgdg Release.gpg
Get:8 http://us-central1.gce.archive.ubuntu.com precise/restricted Sources [5,470 B]
Hit http://downloads-distro.mongodb.org dist Release.gpg
Get:9 http://security.ubuntu.com precise-security Release.gpg [198 B]
Get:10 http://us-central1.gce.archive.ubuntu.com precise/universe Sources [5,019 kB]
Hit http://apt.postgresql.org precise-pgdg Release
Get:11 http://security.ubuntu.com precise-security Release [55.5 kB]
Hit http://downloads-distro.mongodb.org dist Release
Hit http://apt.postgresql.org precise-pgdg/main amd64 Packages
Hit http://apt.postgresql.org precise-pgdg/main i386 Packages
Hit http://ppa.launchpad.net precise Release.gpg
Get:12 http://security.ubuntu.com precise-security/main Sources [142 kB]
Hit http://downloads-distro.mongodb.org dist/10gen amd64 Packages
Ign http://apt.postgresql.org precise-pgdg/main TranslationIndex
Get:13 http://us-central1.gce.archive.ubuntu.com precise/multiverse Sources [155 kB]
Get:14 http://us-central1.gce.archive.ubuntu.com precise/main amd64 Packages [1,273 kB]
Hit http://downloads-distro.mongodb.org dist/10gen i386 Packages
Get:15 http://security.ubuntu.com precise-security/restricted Sources [4,476 B]
Get:16 http://us-central1.gce.archive.ubuntu.com precise/restricted amd64 Packages [8,452 B]
Get:17 http://us-central1.gce.archive.ubuntu.com precise/universe amd64 Packages [4,786 kB]
Hit http://ppa.launchpad.net precise Release.gpg
Get:18 http://security.ubuntu.com precise-security/universe Sources [48.8 kB]
Ign http://downloads-distro.mongodb.org dist/10gen TranslationIndex
Get:19 http://security.ubuntu.com precise-security/multiverse Sources [2,731 B]
Get:20 http://security.ubuntu.com precise-security/main amd64 Packages [626 kB]
Ign http://apt.postgresql.org precise-pgdg/main Translation-en_US
Hit http://ppa.launchpad.net precise Release.gpg
Get:21 http://us-central1.gce.archive.ubuntu.com precise/multiverse amd64 Packages [119 kB]
Get:22 http://us-central1.gce.archive.ubuntu.com precise/main i386 Packages [1,274 kB]
Ign http://apt.postgresql.org precise-pgdg/main Translation-en
Get:23 http://us-central1.gce.archive.ubuntu.com precise/restricted i386 Packages [8,431 B]
Get:24 http://us-central1.gce.archive.ubuntu.com precise/universe i386 Packages [4,796 kB]
Get:25 http://security.ubuntu.com precise-security/restricted amd64 Packages [11.6 kB]
Get:26 http://security.ubuntu.com precise-security/universe amd64 Packages [133 kB]
Hit http://ppa.launchpad.net precise Release.gpg
Get:27 http://security.ubuntu.com precise-security/multiverse amd64 Packages [3,161 B]
Get:28 http://security.ubuntu.com precise-security/main i386 Packages [691 kB]
Get:29 http://us-central1.gce.archive.ubuntu.com precise/multiverse i386 Packages [121 kB]
Get:30 http://ppa.launchpad.net precise Release.gpg [316 B]
Get:31 http://security.ubuntu.com precise-security/restricted i386 Packages [11.6 kB]
Get:32 http://us-central1.gce.archive.ubuntu.com precise/main TranslationIndex [3,706 B]
Get:33 http://us-central1.gce.archive.ubuntu.com precise/multiverse TranslationIndex [2,676 B]
Get:34 http://us-central1.gce.archive.ubuntu.com precise/restricted TranslationIndex [2,596 B]
Get:35 http://us-central1.gce.archive.ubuntu.com precise/universe TranslationIndex [2,922 B]
Get:36 http://us-central1.gce.archive.ubuntu.com precise-updates/main Sources [497 kB]
Get:37 http://security.ubuntu.com precise-security/universe i386 Packages [141 kB]
Get:38 http://us-central1.gce.archive.ubuntu.com precise-updates/restricted Sources [8,708 B]
Get:39 http://us-central1.gce.archive.ubuntu.com precise-updates/universe Sources [128 kB]
Get:40 http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse Sources [10.2 kB]
Get:41 http://us-central1.gce.archive.ubuntu.com precise-updates/main amd64 Packages [1,004 kB]
Get:42 http://security.ubuntu.com precise-security/multiverse i386 Packages [3,335 B]
Get:43 http://us-central1.gce.archive.ubuntu.com precise-updates/restricted amd64 Packages [16.2 kB]
Get:44 http://us-central1.gce.archive.ubuntu.com precise-updates/universe amd64 Packages [277 kB]
Get:45 http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse amd64 Packages [16.9 kB]
Hit http://ppa.launchpad.net precise Release.gpg
Get:46 http://us-central1.gce.archive.ubuntu.com precise-updates/main i386 Packages [1,073 kB]
Get:47 http://security.ubuntu.com precise-security/main TranslationIndex [208 B]
Get:48 http://us-central1.gce.archive.ubuntu.com precise-updates/restricted i386 Packages [16.1 kB]
Get:49 http://us-central1.gce.archive.ubuntu.com precise-updates/universe i386 Packages [287 kB]
Get:50 http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse i386 Packages [17.1 kB]
Get:51 http://us-central1.gce.archive.ubuntu.com precise-updates/main TranslationIndex [208 B]
Get:52 http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse TranslationIndex [202 B]
Hit http://ppa.launchpad.net precise Release.gpg
Get:53 http://us-central1.gce.archive.ubuntu.com precise-updates/restricted TranslationIndex [202 B]
Get:54 http://us-central1.gce.archive.ubuntu.com precise-updates/universe TranslationIndex [205 B]
Get:55 http://us-central1.gce.archive.ubuntu.com precise-backports/main Sources [5,922 B]
Ign http://downloads-distro.mongodb.org dist/10gen Translation-en_US
Get:56 http://us-central1.gce.archive.ubuntu.com precise-backports/restricted Sources [28 B]
Get:57 http://us-central1.gce.archive.ubuntu.com precise-backports/universe Sources [44.2 kB]
Get:58 http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse Sources [5,750 B]
Get:59 http://security.ubuntu.com precise-security/multiverse TranslationIndex [199 B]
Get:60 http://us-central1.gce.archive.ubuntu.com precise-backports/main amd64 Packages [6,477 B]
Get:61 http://us-central1.gce.archive.ubuntu.com precise-backports/restricted amd64 Packages [28 B]
Get:62 http://us-central1.gce.archive.ubuntu.com precise-backports/universe amd64 Packages [46.2 kB]
Get:63 http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse amd64 Packages [5,419 B]
Get:64 http://us-central1.gce.archive.ubuntu.com precise-backports/main i386 Packages [6,478 B]
Get:65 http://us-central1.gce.archive.ubuntu.com precise-backports/restricted i386 Packages [28 B]
Get:66 http://us-central1.gce.archive.ubuntu.com precise-backports/universe i386 Packages [46.1 kB]
Get:67 http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse i386 Packages [5,413 B]
Get:68 http://us-central1.gce.archive.ubuntu.com precise-backports/main TranslationIndex [202 B]
Get:69 http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse TranslationIndex [202 B]
Get:70 http://us-central1.gce.archive.ubuntu.com precise-backports/restricted TranslationIndex [193 B]
Get:71 http://us-central1.gce.archive.ubuntu.com precise-backports/universe TranslationIndex [205 B]
Hit https://packagecloud.io precise Release.gpg
Get:72 http://security.ubuntu.com precise-security/restricted TranslationIndex [202 B]
Ign http://downloads-distro.mongodb.org dist/10gen Translation-en
Get:73 http://us-central1.gce.archive.ubuntu.com precise/main Translation-en [726 kB]
Get:74 http://security.ubuntu.com precise-security/universe TranslationIndex [205 B]
Hit http://ppa.launchpad.net precise Release
Get:75 http://us-central1.gce.archive.ubuntu.com precise/multiverse Translation-en [93.4 kB]
Get:76 http://us-central1.gce.archive.ubuntu.com precise/restricted Translation-en [2,395 B]
Get:77 http://us-central1.gce.archive.ubuntu.com precise/universe Translation-en [3,341 kB]
Get:78 http://security.ubuntu.com precise-security/main Translation-en [259 kB]
Get:79 http://security.ubuntu.com precise-security/multiverse Translation-en [1,698 B]
Get:80 http://us-central1.gce.archive.ubuntu.com precise-updates/main Translation-en [420 kB]
Get:81 http://security.ubuntu.com precise-security/restricted Translation-en [2,976 B]
Get:82 http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse Translation-en [9,806 B]
Get:83 http://us-central1.gce.archive.ubuntu.com precise-updates/restricted Translation-en [3,869 B]
Hit http://ppa.launchpad.net precise Release
Get:84 http://us-central1.gce.archive.ubuntu.com precise-updates/universe Translation-en [166 kB]
Get:85 http://us-central1.gce.archive.ubuntu.com precise-backports/main Translation-en [5,737 B]
Get:86 http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse Translation-en [4,852 B]
Get:87 http://us-central1.gce.archive.ubuntu.com precise-backports/restricted Translation-en [28 B]
Hit https://packagecloud.io precise Release
Get:88 http://us-central1.gce.archive.ubuntu.com precise-backports/universe Translation-en [35.9 kB]
Get:89 http://security.ubuntu.com precise-security/universe Translation-en [85.0 kB]
Hit http://ppa.launchpad.net precise Release
Hit https://packagecloud.io precise/main amd64 Packages
Hit http://ppa.launchpad.net precise Release
Get:90 http://ppa.launchpad.net precise Release [12.9 kB]
Hit https://packagecloud.io precise/main i386 Packages
Get:91 https://packagecloud.io precise/main TranslationIndex [162 B]
Ign https://packagecloud.io precise/main TranslationIndex
Hit http://ppa.launchpad.net precise Release
Hit http://ppa.launchpad.net precise Release
Hit http://ppa.launchpad.net precise/main amd64 Packages
Ign https://packagecloud.io precise/main Translation-en_US
Hit http://ppa.launchpad.net precise/main i386 Packages
Ign https://packagecloud.io precise/main Translation-en
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main Sources
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Get:92 http://ppa.launchpad.net precise/main amd64 Packages [1,398 B]
Get:93 http://ppa.launchpad.net precise/main i386 Packages [943 B]
Get:94 http://ppa.launchpad.net precise/main TranslationIndex [199 B]
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Get:95 http://ppa.launchpad.net precise/main Translation-en [735 B]
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Fetched 29.2 MB in 7s (3,762 kB/s)
Reading package lists...
Hit http://us-central1.gce.archive.ubuntu.com precise Release.gpg
Hit http://us-central1.gce.archive.ubuntu.com precise-updates Release.gpg
Hit http://us-central1.gce.archive.ubuntu.com precise-backports Release.gpg
Hit http://us-central1.gce.archive.ubuntu.com precise Release
Hit http://us-central1.gce.archive.ubuntu.com precise-updates Release
Hit http://us-central1.gce.archive.ubuntu.com precise-backports Release
Hit http://us-central1.gce.archive.ubuntu.com precise/main Sources
Hit http://us-central1.gce.archive.ubuntu.com precise/restricted Sources
Hit http://us-central1.gce.archive.ubuntu.com precise/universe Sources
Hit http://us-central1.gce.archive.ubuntu.com precise/multiverse Sources
Hit http://us-central1.gce.archive.ubuntu.com precise/main amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise/restricted amd64 Packages
Hit http://apt.postgresql.org precise-pgdg Release.gpg
Hit http://us-central1.gce.archive.ubuntu.com precise/universe amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise/multiverse amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise/main i386 Packages
Hit http://downloads-distro.mongodb.org dist Release.gpg
Hit http://security.ubuntu.com precise-security Release.gpg
Hit http://apt.postgresql.org precise-pgdg Release
Hit http://us-central1.gce.archive.ubuntu.com precise/restricted i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise/universe i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise/multiverse i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise/main TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise/multiverse TranslationIndex
Hit http://security.ubuntu.com precise-security Release
Hit http://us-central1.gce.archive.ubuntu.com precise/restricted TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise/universe TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/main Sources
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/restricted Sources
Hit http://downloads-distro.mongodb.org dist Release
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/universe Sources
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse Sources
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/main amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/restricted amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/universe amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse amd64 Packages
Hit http://apt.postgresql.org precise-pgdg/main amd64 Packages
Hit http://security.ubuntu.com precise-security/main Sources
Hit http://apt.postgresql.org precise-pgdg/main i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/main i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/restricted i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/universe i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/main TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/restricted TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/universe TranslationIndex
Hit http://security.ubuntu.com precise-security/restricted Sources
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/main Sources
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/restricted Sources
Hit http://downloads-distro.mongodb.org dist/10gen amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/universe Sources
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse Sources
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/main amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/restricted amd64 Packages
Hit http://ppa.launchpad.net precise Release.gpg
Ign http://apt.postgresql.org precise-pgdg/main TranslationIndex
Hit http://security.ubuntu.com precise-security/universe Sources
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/universe amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/main i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/restricted i386 Packages
Hit http://downloads-distro.mongodb.org dist/10gen i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/universe i386 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse i386 Packages
Hit http://security.ubuntu.com precise-security/multiverse Sources
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/main TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/restricted TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/universe TranslationIndex
Hit http://us-central1.gce.archive.ubuntu.com precise/main Translation-en
Hit http://us-central1.gce.archive.ubuntu.com precise/multiverse Translation-en
Hit http://us-central1.gce.archive.ubuntu.com precise/restricted Translation-en
Hit http://us-central1.gce.archive.ubuntu.com precise/universe Translation-en
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/main Translation-en
Hit http://security.ubuntu.com precise-security/main amd64 Packages
Ign http://downloads-distro.mongodb.org dist/10gen TranslationIndex
Hit http://ppa.launchpad.net precise Release.gpg
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/multiverse Translation-en
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/restricted Translation-en
Hit http://us-central1.gce.archive.ubuntu.com precise-updates/universe Translation-en
Hit http://security.ubuntu.com precise-security/restricted amd64 Packages
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/main Translation-en
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/multiverse Translation-en
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/restricted Translation-en
Hit http://us-central1.gce.archive.ubuntu.com precise-backports/universe Translation-en
Hit http://security.ubuntu.com precise-security/universe amd64 Packages
Hit http://security.ubuntu.com precise-security/multiverse amd64 Packages
Hit http://ppa.launchpad.net precise Release.gpg
Hit http://security.ubuntu.com precise-security/main i386 Packages
Ign http://apt.postgresql.org precise-pgdg/main Translation-en_US
Ign http://apt.postgresql.org precise-pgdg/main Translation-en
Hit http://security.ubuntu.com precise-security/restricted i386 Packages
Hit http://security.ubuntu.com precise-security/universe i386 Packages
Hit http://security.ubuntu.com precise-security/multiverse i386 Packages
Hit http://ppa.launchpad.net precise Release.gpg
Hit http://security.ubuntu.com precise-security/main TranslationIndex
Hit http://ppa.launchpad.net precise Release.gpg
Hit http://security.ubuntu.com precise-security/multiverse TranslationIndex
Hit http://security.ubuntu.com precise-security/restricted TranslationIndex
Hit http://security.ubuntu.com precise-security/universe TranslationIndex
Hit http://ppa.launchpad.net precise Release.gpg
Hit http://security.ubuntu.com precise-security/main Translation-en
Hit http://security.ubuntu.com precise-security/multiverse Translation-en
Hit http://security.ubuntu.com precise-security/restricted Translation-en
Hit http://ppa.launchpad.net precise Release.gpg
Ign http://downloads-distro.mongodb.org dist/10gen Translation-en_US
Hit http://security.ubuntu.com precise-security/universe Translation-en
Ign http://downloads-distro.mongodb.org dist/10gen Translation-en
Hit http://ppa.launchpad.net precise Release
Hit http://ppa.launchpad.net precise Release
Hit http://ppa.launchpad.net precise Release
Hit https://packagecloud.io precise Release.gpg
Hit http://ppa.launchpad.net precise Release
Hit http://ppa.launchpad.net precise Release
Hit https://packagecloud.io precise Release
Hit http://ppa.launchpad.net precise Release
Hit http://ppa.launchpad.net precise Release
Hit https://packagecloud.io precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit https://packagecloud.io precise/main i386 Packages
Get:1 https://packagecloud.io precise/main TranslationIndex [162 B]
Ign https://packagecloud.io precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Ign https://packagecloud.io precise/main Translation-en_US
Hit http://ppa.launchpad.net precise/main TranslationIndex
Ign https://packagecloud.io precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Sources
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main amd64 Packages
Hit http://ppa.launchpad.net precise/main i386 Packages
Hit http://ppa.launchpad.net precise/main TranslationIndex
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Hit http://ppa.launchpad.net precise/main Translation-en
Reading package lists...
The command "sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install mysql-server-5.6 mysql-client-core-5.6 mysql-client-5.6" failed and exited with 100 during .
``` 

😢

**Comment by vladikoff** [comment]
from Travis:

```
Update
We've determined that an image cleanup process malfunctioned and deleted the stable build environment images. Due to various factors our only option at this point is roll forward with newer build environment images, for both Precise and Trusty. We're resuming builds with these new images, but these new images have not undergone much testing, so it is possible your builds will break. If you see any issues, please email support at travis-ci.com, our engineering staff is available to help with any issues you run into. We'll also be investigating the exact cause of the issue and will be publishing a postmortem as we learn more.
Posted about 1 hour ago. Aug 09, 2016 - 15:50 UTC
```

**Comment by vladikoff** [comment]
ok it was fixed! 👍

**Pull request merged**